src/integTest/resources/sample.servlet/testOutputDirs.gradle

```diff
@@ -0,0 +1,116 @@
+group = 'liberty.gradle'
+version = '1'
+
+buildscript {
+    repositories {
+        mavenLocal()
+        mavenCentral()
+        maven {
+            name = 'Sonatype Nexus Snapshots'
+            url = 'https://oss.sonatype.org/content/repositories/snapshots/'
+        }
+    }
+    dependencies {
+        classpath 'net.wasdev.wlp.gradle.plugins:liberty-gradle-plugin:1.1-SNAPSHOT'
+    }
+}
+
+apply plugin: 'maven'
+apply plugin: 'war'
+apply plugin: 'liberty'
+
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
+
+compileJava.options.encoding = 'UTF-8'
+
+ext {
+    // Liberty server properties
+    wlpServerName = 'LibertyProjectServer'
+    serverDirectory = "${project.buildDir}/wlp/usr/servers/${wlpServerName}"
+    testServerHttpPort = 9080
+    testServerHttpsPort = 9443
+
+    warContext = 'sample.servlet-1'
+    appName = 'LibertyProject'
+    packageFile = "${project.buildDir}/${appName}.zip"
+    packagingType = 'usr'
+}
+
+task testWar(type:War){
+    baseName = 'testWar'
+}
+
+task createServerEnv{
+    def serverEnvFile = file("src/main/liberty/config/testOutputDir-server.env")
+    if (!serverEnvFile.exists()) {
+        serverEnvFile.write("WLP_OUTPUT_DIR=${project.buildDir}/testOutputDir")
+    }
```

Comment:
At first, I was wondering if this file could have some residual effect on the next test, but since it isn't names server.env, it should be ok.  In fact, I think this file could just be configured in the source project rather than created dynamically if that is easier.